### ISSUE TITLE
Start minimized

### ISSUE THREAD

**Issue by nitin2953** [U-C-S/Hurl#49]
I have kept Hurl in Startup folder,
On every reboot and power up it shows on top of other widows & I have to minimise it

**Comment by U-C-S** [comment]
yep.... I tried implementing it a while ago. Had some issues then, so it's partially done

https://github.com/U-C-S/Hurl/blob/535e419fcfb79a6362d9e945916eb055f6cdb58f/App/Hurl.BrowserSelector/Helpers/CliArgs.cs#L10

...I will give a chance again

**Comment by U-C-S** [comment]
Looks easy enough to have this. but was also wondering if there would be any use on starting this ahead.

> usually take clicking at least twice on a hyperlink for Hurl to actually open.

If this is the only reason, then I would just fix the clicking twice issue. but Its little tricky as of now to debug this one. Nevertheless will think about it.
